workspaces/website/src/components/Layout/PageLayout.tsx

```diff
@@ -83,6 +101,126 @@ export const PageLayout = (props: Props) => {
+            overflow="hidden"
+          >
+            <Box
+              sx={{
+                background: "linear-gradient(170deg, #0C0C4F 4.53%, #0C0C4F 45.92%, #6170B5E6 80.48%, #A47DAAE6 93.63%, #F99B84CC 100%)",
+              }}
+            >
+              <Box
+                sx={{
+                  backgroundImage: "/assets/SNSummitTower.svg",
+                  backgroundRepeat: "no-repeat",
+                  backgroundSize: "332px",
+                  backgroundPosition: "0 70px",
+                  borderTopLeftRadius: "8px",
+                  borderTopRightRadius: "8px",
+                  display: "flex",
+                  alignItems: "center",
+                  flexDirection: "column",
+                  height: "220px",
+                  pt: "51px"
+                }}
+              >
+                <Icon as={IoCloseOutline}
+                  width="28px"
+                  height="28px"
+                  position='absolute'
+                  right="18px"
+                  top="18px"
+                  cursor="pointer"
+                  color="white"
+                  onClick={closePopup}
+                />
+                <img
+                  src="/assets/SummitLogo.svg"
+                  alt="Summit"
+                />
+              </Box>
+            </Box>
+            <Box
+              background="#0C0C4F"
+              flex="1"
+              p="24px"
+              width="332px"
+              borderBottomLeftRadius="8px"
+              borderBottomRightRadius="8px"
+            >
+              <Flex
+                direction="row"
+                justifyContent="space-between"
+                pb="16px"
+              >
+                <Heading
+                  variant="h4"
+                  fontWeight="400"
+                  color="white"
+                >
+                  San Francisco
+                </Heading>
+                <Heading
+                  variant="h4"
+                  fontWeight="400"
+                  color="summit-popup-date-heading-color"
+                  sx={{
+                    background: "-webkit-linear-gradient(45deg, #EC796B, #D672EF 80%)",
+                    "-webkit-background-clip": "text",
+                    "-webkit-text-fill-color": "transparent",
+                    paddingLeft: "24px",
+                    borderLeft: "1px solid #EC796B",
+                    marginLeft: "20px"
+                  }}
+                >
+                  Aug 31st 2023
+                </Heading>
+              </Flex>
+              <Button
+                variant="outline"
+                sx={{
+                  background: "white",
+                  width: "100%",
+                  color: "#0C0C4F",
+                  _dark: {
+                    color: "#0C0C4F"
+                  }
+                }}
+                href="https://summit23.starknet.io/"
+                target="_blank"
+              >
+                Get Tickets
```

Comment:
@milosyuki can you extract this to its own component including its states? It will be easier to remove it once promotion is over and it keeps PageLayout cleaner.